ballerina/build.gradle

```diff
@@ -725,7 +725,6 @@ task testExamples() {
             'retry-transactions',
             //TODO: Enable following for slp4
             'iterable-objects',
-            'range-expressions',
```

Comment:
Shall we update the BBE without this change until the build picks the updated lang version?